export function error(msg) {
    try {
        console.error('error:', msg);
    } catch(e) {}
}
